### ISSUE TITLE
Publish prebuild binaries

### ISSUE THREAD

**Issue by felixfbecker** [node-inspector/v8-profiler#119]
Compiling v8-profiler currently takes up a significant part of install & total build time in CI. I don't know if this is possible, but it would be great if pre-built binaries for the usual targets were uploaded to GitHub with [`prebuild`](https://www.npmjs.com/package/prebuild) and then defaulted to if available with [`prebuild-install`](https://github.com/prebuild/prebuild-install) (falling back to compiling like today).

**Comment by linxuanwei** [comment]
agree

**Comment by egorvoronov** [comment]
yeah, absense of prebuilt binaries also requires xcode to be installed on mac and xcode installation requires admin rights that could be the problem for teammates who don't have such.
